doc/motr-in-prose.md

```diff
@@ -225,7 +225,7 @@ re-ordering. See NRS in the references below.
 Each node in the cluster runs one or more Motr instances. Each instance has the 
 same overall structure:
 
-![Motr Instance](https://github.com/Seagate/cortx-motr/blob/Motr_Desc-Guide/doc/Images/2-Mero-Instance.JPG)
+![](/doc/Images/2-Mero-Instance.JPG)
```

Comment:
Rename this image to 2-Motr-Instance.JPG

Reply:
Changed. Please approve @madhavemuri